edx_notifications/server/web/templates/django/digests/branded_notifications_outer.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title></title>
+</head>
+<body>
+<div class="xns-container">
+    <div class="xns-content xns-pane-digest">
+        <div class="xns-list-header">
+            <div class="xns-logos" style="margin-bottom:20px;">
+                <div class="xns-open-edx-logo"><img src="cid:{{open_edx_default_logo}}" width="100" height="50"></div>
+                <div class="xns-branded-logo"> <img src="cid:{{branded_default_logo}}" width="100" height="50"></div>
```

Comment:
What happens if we don't constrain the width and height? I'm not sure we'll be able to force people to use 100x50 logos....